conans/client/migrations_settings.py

```diff
@@ -911,3 +911,94 @@
 settings_1_22_3 = settings_1_22_2
 
 settings_1_23_0 = settings_1_22_3
+settings_1_24_0 = """
```

Comment:
This is not the right place to change the settings.

please change it in ``conans/client/conf/__init__.py``

We will update the migration as necessary.